.github/workflows/dockerhub-description.yml

```diff
@@ -0,0 +1,23 @@
+name: Update Docker Hub Description
+permissions:
+  contents: read
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - README.md
+      - .github/workflows/dockerhub-description.yml
+jobs:
+  dockerHubDescription:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Docker Hub Description
+      uses: peter-evans/dockerhub-description@1b9a80c056b620d92cedb9d9b5a223409c68ddfa #v5
+      with:
+        username: ${{ secrets.DOCKERHUB_USER }}
+        password: ${{ secrets.DOCKERHUB_PASS }}
+        repository: pihole/pihole
+        short-description: ${{ github.event.repository.description }}
```

Comment:
Currently, `github.event.repository.description` is 'Pi-hole in a docker container', but the docker description is `The official Pi-hole Docker image from pi-hole.net`.

I thin the one we use currently on docker hub should stay.